elasticsearch/run.sh

```diff
@@ -2,12 +2,23 @@
 
 set -euo pipefail
 
-mkdir -p /elasticsearch/$CLUSTER_NAME
-secret_dir=/etc/elasticsearch/secret/
-[ -f $secret_dir/searchguard-node-key ] && ln -s $secret_dir/searchguard-node-key /elasticsearch/$CLUSTER_NAME/searchguard_node_key.key
-[ -f $secret_dir/searchguard.key ] && ln -s $secret_dir/searchguard.key /elasticsearch/$CLUSTER_NAME/searchguard_node_key.key
-[ -f $secret_dir/keystore.password ] && export KEYSTORE_PASSWORD=$(cat $secret_dir/keystore.password)
-[ -f $secret_dir/truststore.password ] && export TRUSTSTORE_PASSWORD=$(cat $secret_dir/truststore.password)
+secret_dir=/etc/elasticsearch/secret
+config_dir=/etc/elasticsearch/config
+
+# ES seems to be picky about its config/ dir and permissions within it -- ln -s would yield incorrect permissions...
```

Comment:
it seems that we were able to do things like that before in 1.5 since the java security manager wasn't enabled.  any objections to running with it disabled in 2.x?